cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -926,7 +926,12 @@
         " bar "
         " )"]
        {:remove-surrounding-whitespace? false
-        :remove-trailing-whitespace? false})))
+        :remove-trailing-whitespace? false}))
+  (is (reformats-to?
+       ["{:one two :three four}"]
```

Comment:
This needs more tests to check edge conditions. What happens with comments, with commas, with `#_`, etc.